docs/spark-standalone.md

```diff
@@ -24,7 +24,7 @@ the master's web UI, which is [http://localhost:8080](http://localhost:8080) by
 
 Similarly, you can start one or more workers and connect them to the master via:
 
-    ./bin/spark-class org.apache.spark.deploy.worker.Worker spark://IP:PORT
+    ./sbin/start-slave.sh <worker#> <master-spark-URL>
```

Comment:
if we run `./sbin/start-slave.sh <worker#> <master-spark-URL>`, the data flow would be:
* in start-slave.sh
"$sbin"/spark-daemon.sh start org.apache.spark.deploy.worker.Worker <worker#> <master-spark-URL>

* in spark-daemon.sh
----->
option=start command=org.apache.spark.deploy.worker.Worker instance=<worker#>
run_command class <master-spark-URL>
----->
nohup nice -n "$SPARK_NICENESS" "$SPARK_PREFIX"/bin/spark-class org.apache.spark.deploy.worker.Worker <master-spark-URL> >> "$log" 2>&1 < /dev/null &

So we can tell except once more `load-spark-env.sh`(also is done in `spark-class`) and log/backgroud handle process, these two ways are same actually.